frontend/Android/whereToStart.md

```diff
@@ -0,0 +1,17 @@
+<div dir="rtl">
+  
+# Android development. 
+
+# where to start
+- محتاج ايه علشان ابدا اتعلم اندرويد؟
+
+ > تعلم الاندرويد مش او حاجه تتعلمها علشان تتعلم اندرويد لازم تتعلم برمجه كويسه وتكون فاهم ازاي تبني برنامج باستخدام اي لغه برمجه وتكون فاهم  OOP (  Object Oriented Programming) و ممكن تتعلم data structures and algorithms  علشان تبقي مبرمج احسن 
+
+> الاندوريد حاليا قايم علي جزئين جزء واجهه المستخدم وهو  UI ( User interface) وده بيتم في الاندرويد عن طريق لغه اسمها xml  ودي شبهه  HTML  والجزء التاني الخاص بالبرمجه نفسها وبيتم بلغتين حاليا وهما kotlin and java 
+حاليا كل الشركات والمشاريع الجديده بدآت ب kotlin فيستحسن لو هتدخل المجال جديد تدخل تتعلم kotlin لان المشاريع اللي شغاله ب java معظمها مشاريع قديمه وبيحصل عليها  maintaing
+
+> من احسن الكورسات الموجوده دلوقتي كورس جوجل للاندرويد علي udacity 
+> [Android Nano Degree](https://www.udacity.com/course/android-developer-nanodegree-by-google--nd801) 
+
+---
+  <div dir="rtl">
```

Comment:
you just need to end the div section, I've modified it since it was wrong in the first file
```suggestion
  </div>
```